src/lib/codeSamples/buildRequest.ts

```
@@ -0,0 +1,109 @@
+
+function getPath(variables, pathParameters, path) {
+  for (const [key, value] of Object.entries(variables)) {
+    if (!pathParameters.find(parameter => parameter.name === key)) {
+      continue
+    }
+
+    if (value === undefined || value === '') {
+      continue
+    }
+
+    path = path.replace(`{${key}}`, value)
```

Biome

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)